mgit/README.md

```diff
@@ -0,0 +1,87 @@
+# MGit
+
+Tool for management the multi-repository architecture.
+
+## Instalation
+
+```bash
+npm install @cksource/mgit
+```
+
+## Usage
+
+Firstly, you need to create a configuration file `mgit.json`:
+
+```json
+{
+  "packages": "packages/",
+  "dependencies": {
+    "package/path": "github/url"
+  }
+}
+```
+
+* `packages` - the directory where dependencies will be cloned,
+* `dependencies` - list of dependencies.
+
+The following dependencies are proper and can be placed into configuration file:
+
+Package will be installed in `packages/@scope/package` from branch `develop`:
+
+```json
+{
+	"@scope/package": "scopes/package#develop"
+}
+```
+
+Package will be installed in `packages/package` from branch `master`:
+
+```json
+{
+	"package": "https://github.com/github/package"
```

Comment:
It does not matter because the `bootstrap` command creates a command to clone a repository manually:

```js
const command = `git clone -b ${ parsedRepository.branch } git@github.com:${ parsedRepository.repository }.git ${ destinationPath }`;
```

The configuration has to provide correct URLs, `bootstrap` will build correct git command and everything will work.